### PR TITLE
Home task:HTML 01 first part

### PR DESCRIPTION
**Pull request by Artem781**

### CODE REVIEW

File: anaron/Decision_1.2.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    <h2>Пример цикла For</h2>
+    <p>Теперь давайте рассмотрим пример программирования цикла. Запустите текстовый
+        редактор и наберите следующую программу:</p>
+    <pre>#!/usr/bin/env python
+for cntr in range(0,10):
+    print cntr</pre>
+    <p><em>Не забудьте поставить символ табуляции перед строкой «print cntr».</em> Это важно. В
+        Python для указания на блоки кода используются не круглые «(» и фигурные «{» скобки
+        как в других языках программирования, а отступы.</p>
+
```
Comment (pluhin) [inline anaron/Decision_1.2.html]: Need to remove this extra lines

File: anaron/Decision_1.3.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ,
+        комментарии станут важными. Также при их помощи вы заставите Python игнорировать
+        какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'.
+        Например:</p>
+    #
+    <p>Это комментарий
+    </p>
+    <p>Комментарии можно вставлять в любое место в строке, но помните, что Python
+        проигнорирует всё, что идёт после '#'.</p>
+
```
Comment (pluhin) [inline anaron/Decision_1.3.html]: Empty lines  nee d remove

File: anaron/Decision_1.3.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        списков.</p>
+    <hr>
+    <h2>Комментарии</h2>
+    <p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам.
+        Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят
+        вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ,
+        комментарии станут важными. Также при их помощи вы заставите Python игнорировать
+        какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'.
+        Например:</p>
+    #
```
Comment (pluhin) [inline anaron/Decision_1.3.html]: \# - also code, need to shift into pre tag

File: anaron/Decision_1.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    <!--заголовок страницы-->
+    <TITLE>Пример страницы</TITLE>
+</HEAD>
+<!-- начало ддокумента-->
+
+<BODY>
+    <H1>Это заголовок</H1>
+    <P><DEL>Добрый день, <EM>Уважаемые слушатели</EM></DEL>! Сегодня мы с вами будем рассматривать структуру стандартного <STRONG>HTML</STRONG> файла.</P>
+    <P>Далее мы будем использовать произвольный текст для наполнения контентом. Не стоит вдумываться в его содержание.</P>
+    <P>«Си́мпсоны» (англ. The Simpsons) — самый длинный мультсериал в истории американского телевидения, длящийся 25 сезонов. Первая мини-серия «Good night» была показана в «Шоу Трейси Ульман» 19 апреля 1987 года.
```
Comment (pluhin) [inline anaron/Decision_1.html]: This tag p wasn't closed

**Comment by Artem781** [comment]
decision_1-fixed- In lines №15 I closed tag P(</P>),
decision_1.2-fixed-In lines №54 I deleted empty lines,
decision_1.3-fixed-In lines №53 i put element # in tag <pre>,
I corrected task accoding to your comments. Please cheak it

**Comment by Artem781** [comment]
decision_1-fixed- In lines №15 I closed tag P(</P>),
decision_1.2-fixed-In lines №54 I deleted empty lines,
decision_1.3-fixed-In lines №53 i put element # in tag <pre>,
I corrected task accoding to your comments. Please cheak it

File: anaron/02. HTML. Table/Decision_2_PytonPart3.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<thml>
+    <head>
+    </head>
+    <meta charset="UTF-8">
```
Comment (pluhin) [inline anaron/02. HTML. Table/Decision_2_PytonPart3.html]: this two tags should be inside tag head

Comment (Artem781) [reply anaron/02. HTML. Table/Decision_2_PytonPart3.html]: thank. I corrected this error.

File: anaron/02. HTML. Table/site.by/about.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+<HEAD>
+    <meta charset="utf-8">
+    <TITLE>Пример страницы</TITLE>
+</HEAD>
+
+<BODY>
+    <H1>about</H1>
+    <h2>Меню с вложенным списком</h2>
+    <ul>
+        <li><a href="about.html">about</a></li>
```
Comment (pluhin) [inline anaron/02. HTML. Table/site.by/about.html]: this li tag should be contain whole sub list, not just tag a, look carefully our slides

File: anaron/02. HTML. Table/site.by/about.html [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+        <ul>
+            <li><a href="about/contacts.html">contacts</a></li>
+            <li><a href="about/scheme.html">scheme</a></li>
+        </ul>
+        <li><a href="catalog.html">catalog</a></li>
```
Comment (pluhin) [inline anaron/02. HTML. Table/site.by/about.html]: same as above

File: anaron/02. HTML. Table/site.by/about.html [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+        <ul>
+            <li><a href="catalog/a_w.html">a_w</a></li>
+            <li><a href="catalog/s_s.html">s_s</a></li>
+        </ul>
+        <li><a href="news/news.html">news</a></li>
```
Comment (pluhin) [inline anaron/02. HTML. Table/site.by/about.html]: same as above

File: anaron/02. HTML. Table/site.by/about.html [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+        <ul>
+            <li><a href="news/news.html">news</a></li>
+        </ul>
+        <li><a href="promo.html">promo</a></li>
```
Comment (pluhin) [inline anaron/02. HTML. Table/site.by/about.html]: same as above

File: anaron/02. HTML. Table/site.by/about/contacts.html [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+<HEAD>
+    <meta charset="utf-8">
+    <TITLE>Пример страницы</TITLE>
+</HEAD>
+
+<BODY>
+    <H1>contacts</H1>
+    <ul>
+        <li><a href="../about.html">about</a></li>
```
Comment (pluhin) [inline anaron/02. HTML. Table/site.by/about/contacts.html]: wrong sublist construction

**Pull request merged**